users/vieting/experiments/librispeech/librispeech_100_ctc/fairseq_finetuning/ctc_standalone/default_tools.py

```diff
@@ -0,0 +1,53 @@
+"""
+Defines the external software to be used for the Experiments
+"""
+from sisyphus import tk
+
+from i6_core.tools.git import CloneGitRepositoryJob
+from i6_core.lm.kenlm import CompileKenLMJob
+from i6_experiments.users.vieting.experiments.librispeech.librispeech_960_pretraining.wav2vec2.fairseq \
+    import SetupFairseqJob
+
+from i6_experiments.common.helpers.text_labels.subword_nmt_bpe import get_returnn_subword_nmt
+from i6_experiments.common.tools.sctk import compile_sctk
+
+# python from apptainer/singularity/docker
+RETURNN_EXE = tk.Path("/usr/bin/python3", hash_overwrite="GENERIC_RETURNN_LAUNCHER")
+
+MINI_RETURNN_ROOT = CloneGitRepositoryJob(
+    "https://github.com/JackTemaki/MiniReturnn", commit="0dc69329b21ce0acade4fcb2bf1be0dc8cc0b121"
+).out_repository.copy()
+MINI_RETURNN_ROOT.hash_overwrite = "LIBRISPEECH_DEFAULT_RETURNN_ROOT"
+
+I6_MODELS_REPO_PATH = CloneGitRepositoryJob(
+    url="https://github.com/rwth-i6/i6_models",
+    commit="933c6c13f7d6c74e5a59af0257e17c208dae9da3",
+    checkout_folder_name="i6_models",
+).out_repository.copy()
+I6_MODELS_REPO_PATH.hash_overwrite = "LIBRISPEECH_DEFAULT_I6_MODELS"
+
+
+FAIRSEQ_PATH = SetupFairseqJob(
+    fairseq_root = CloneGitRepositoryJob(
+        url="git@github.com:vieting/fairseq_phoneme.git",
+        checkout_folder_name="fairseq",
+        commit="e4a2e4e93efbcbaaae52a17ae6600beb2083fb33",
+    ).out_repository.copy(),
+    python_exe = "/usr/bin/python3",
+).out_fairseq_root
+
+FAIRSEQ_PATH.hash_overwrite = "LIBRISPEECH_DEFAULT_FAIRSEQ"
```

Comment:
We might need more different versions when experimenting with different pre-training variants, so I think it could be easier to get the fairseq path from a helper in the config file. Similar to the config for pre-training.

Reply:
I guess this is also resolved with the newest commit?